src/backends/kubernetes/kubernetes_backend.py

```diff
@@ -609,11 +632,30 @@ def deploy_pods(self, warnet):
 
             # Create and deploy LND pod
             if tank.lnnode:
-                lnd_container = self.create_lnd_container(tank, bitcoind_service.metadata.name)
+                conts = []
+                vols = []
+                volume_mounts = []
+                if tank.lnnode.cb:
+                    # Create a shared volume between containers in the pod
+                    volume_name = f"ln-cb-data-{tank.index}"
+                    vols.append(client.V1Volume(
+                        name=volume_name,
+                        empty_dir=client.V1EmptyDirVolumeSource()
+                    ))
+                    volume_mounts.append(client.V1VolumeMount(
+                        name=volume_name,
+                        mount_path='/root/.lnd'
```

Comment:
As this is also used in `create_circuitbreaker_container` IMO it makes sense to extract it as a variable at the head of the file?